files/build_templates/docker_image_ctl.j2

```diff
@@ -36,8 +36,11 @@ function updateSyslogConf()
         TARGET_IP=$(docker network inspect bridge --format={{ "'{{(index .IPAM.Config 0).Gateway}}'" }})
         CONTAINER_NAME="$DOCKERNAME"
         TMP_FILE="/tmp/rsyslog.$CONTAINER_NAME.conf"
-
+        {%- if docker_container_name == "database" %} 
+        python -c "import jinja2, os; paths=['/usr/share/sonic/templates']; loader = jinja2.FileSystemLoader(paths); env = jinja2.Environment(loader=loader, trim_blocks=True); template_file='/usr/share/sonic/templates/rsyslog-container.conf.j2'; template = env.get_template(os.path.basename(template_file)); data=template.render({\"target_ip\":\"$TARGET_IP\",\"container_name\":\"$CONTAINER_NAME\"}); print(data)" > $TMP_FILE
+        {%- else %}
```

Comment:
This code is needed only for containers which are attached to namespace. Can we add check to execute this  block only  if the `$DEV` is set?

something like this?

```
 # Also update the container name
    if [[ ($NUM_ASIC -gt 1)  && "$DEV" ]]; then
        TARGET_IP=$(docker network inspect bridge --format='{{(index .IPAM.Config 0).Gateway}}')
        CONTAINER_NAME="$DOCKERNAME"
        TMP_FILE="/tmp/rsyslog.$CONTAINER_NAME.conf"

        sonic-cfggen -t /usr/share/sonic/templates/rsyslog-container.conf.j2 -a "{\"target_ip\": \"$TARGET_IP\", \"container_name\": \"$CONTAINER_NAME\" }"  > $TMP_FILE
        docker cp $TMP_FILE ${DOCKERNAME}:/etc/rsyslog.conf
        rm -rf $TMP_FILE
    fi

```

Reply:
In multi asic platform the container attached to namespace are in a docker bridge network. 
we use `240.127.1.1`(docker bridge ip on the host) rsyslog server address to make sure syslogs generated in containers attached to namespace reaches the host.

For containers running on host, they can use the `127.0.0.1` as the syslog server because they are not in the docker bridge network